AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/CMakeLists.txt

```diff
@@ -122,6 +122,16 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED AND PAL_TRAIT_BUILD_HOST_TOOLS)
             AZ::AssetProcessorBatch
     )
 
+    ly_add_pytest(
+        NAME AssetProcessorCacheServerTests.Periodic
+        PATH  ${CMAKE_CURRENT_LIST_DIR}/asset_processor_cache_server.py
+        #EXCLUDE_TEST_RUN_TARGET_FROM_IDE
+        TEST_SUITE periodic
+        #PYTEST_MARKS "not SUITE_sandbox" # don't run sandbox tests in this file
```

Comment:
Are these 2 lines just comments or do they do something?  Should they be removed?

Reply:
I will add the "exclude from IDE" as it seems to be the pattern 
I am not sure about that sandbox line, it gave me errors when I was iterating on it